content/publications/officer-involved-shootings-in-texas.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Officer-Involved Shootings in Texas: 2016-2019"
+showSidebar: true
+---
+In 2015, Texas lawmakers passed legislation requiring law enforcement agencies to file a simple report within 30 days of an officer-involved shooting, whether fatal or not. The reports began to trickle into the [Office of the Attorney General (OAG)](https://texasattorneygeneral.gov/). The law was amended in 2017 to impose potential penalties – up to a [$1,000-a-day fine](https://capitol.texas.gov/tlodocs/85R/billtext/pdf/HB00245F.pdf#navpanes=0) – for agencies that do not properly file their reports.
+
+This report analyzes data pulled from the officer-involved shooting reports reflecting shootings that took place from Jan. 1, 2016 to Dec. 31, 2019. In this document, TJI also drills into the shootings of civilians by Texas officers, and the shootings of Texas officers to reveal findings about the demography of these shootings – including race – as well as characteristics like where the shootings occur, who is more likely to survive a shooting, and more.
```

Comment:
@nholden @EvaRuthM The second sentence in this paragraph was hard for me to understand. I edited it slightly below, just as a thought.

In this document, TJI also drills into the shootings of civilians by Texas officers, and the shootings of Texas officers by civilians. It reveals findings about the demography of these shootings – including race – as well as characteristics like where the shootings occur, who is more likely to survive a shooting, and more.

Reply:
thanks @LayaTaal! I agree your edit is much more readable than my original. Please do use Jason's text instead, @nholden, with one tweak – please remove "by civilians" at the end of the first sentence. (sometimes they shoot each other)

Reply:
Thanks, @LayaTaal and @EvaRuthM! Will make this change through the CMS after merging.